Makefile

```diff
@@ -43,6 +39,13 @@ IMAGE_REPO ?= quay.io/shipwright
 TAG ?= latest
 IMAGE_PUSH ?= true
 
+# Image URL to use all building/pushing image targets
+IMG ?= $(IMAGE_REPO)/shipwright-operator:$(VERSION)
```

Comment:
`shipwright-operator` is in fact an existing image that contains the build controller.
We probably need to archive it.